ext/pgsql/tests/9999dropdb.phpt

```diff
@@ -3,15 +3,22 @@ PostgreSQL drop db
 --EXTENSIONS--
 pgsql
 --SKIPIF--
-<?php include("skipif.inc"); ?>
+<?php include("inc/skipif.inc"); ?>
 --FILE--
 <?php
 // drop test table
 
-include('config.inc');
+include('inc/config.inc');
+$table_name = "table_9999dropdb";
+$table_name_92 = "table_9999dropdb_92";
+$view_name = "view_9999dropdb";
 
 $db = pg_connect($conn_str);
-pg_query($db, "DROP VIEW {$view_name}");
+pg_query($db, "CREATE TABLE {$table_name} (num int, str text, bin bytea)");
+pg_query($db, "CREATE TABLE {$table_name_92} (textary text[],  jsn json)");
+pg_query($db, "CREATE VIEW {$view_name} as SELECT * FROM {$table_name}");
+
+pg_query($db, "DROP VIEW IF EXISTS {$view_name}");
 pg_query($db, "DROP TABLE ".$table_name);
 @pg_query($db, "DROP TABLE ".$table_name_92);
```

Comment:
```suggestion
pg_query($db, "DROP TABLE IF EXISTS " . $table_name);
pg_query($db, "DROP TABLE IF EXISTS " . $table_name_92);
```
Or maybe use the ``{$var}`` interpolation syntax?

Reply:
Fixed in https://github.com/php/php-src/pull/12608/commits/89aa3f76c30f2444d2f856511d7ddda584fbdba7

Sorry, besides this, I overlooked many other deficiencies in `IF EXISTS`. Modified to include `{$var}` .